transformation/tests/org.jboss.tools.fuse.transformation.editor.tests.integration/src/main/java/org/jboss/tools/fuse/transformation/editor/TransformationEditorIT.java

```diff
@@ -0,0 +1,193 @@
+/******************************************************************************
+ * Copyright (c) 2016 Red Hat, Inc. and others.
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors: JBoss by Red Hat - Initial implementation.
+ *****************************************************************************/
+package org.jboss.tools.fuse.transformation.editor;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.InputStreamReader;
+import java.text.StringCharacterIterator;
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.jface.util.SafeRunnable;
+import org.fusesource.ide.camel.model.service.core.tests.integration.core.io.FuseProject;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class TransformationEditorIT {
+
+	private static final String POM_START = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
+			"<project xsi:schemaLocation=\"http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd\"" +
+			"		  xmlns=\"http://maven.apache.org/POM/4.0.0\"\n" +
+			"    	  xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\">\n" +
+			"  <modelVersion>4.0.0</modelVersion>\n" +
+			"  <groupId>com.mycompany</groupId>\n" +
+			"  <artifactId>testproject</artifactId>\n" +
+			"  <version>1.0.0-SNAPSHOT</version>\n" +
+			"  <packaging>bundle</packaging>\n" +
+			"  <name>Some Dummy Project</name>\n" +
+			"  <build>\n" +
+			"    <plugins>\n";
+
+	private static final String POM_END = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
```

Comment:
first big issue, this line should not exists
"<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"